tests/Predis/Command/RawCommandTest.php

```diff
@@ -74,10 +74,29 @@ public function testExceptionOnMissingCommandID()
      * argument is missing, PHP emits an E_WARNING.
      *
      * @group disconnected
-     * @expectedException \PHPUnit_Framework_Error_Warning
      */
     public function testPHPWarningOnMissingCommandIDWithStaticCreate()
     {
+        if (version_compare(PHP_VERSION, "7.1", '>')) {
+            $this->markTestSkipped('only for PHP < 7.1');
+        }
+        $this->setExpectedException('PHPUnit_Framework_Error_Warning');
+        RawCommand::create();
+    }
+
+    /**
+     * The signature of RawCommand::create() requires one argument which is the
+     * ID of the command (other arguments are fetched dinamically). If the first
+     * argument is missing, PHP 7.1 throw an exception
+     *
+     * @group disconnected
+     */
+    public function testPHPWarningOnMissingCommandIDWithStaticCreate71()
+    {
+        if (version_compare(PHP_VERSION, "7.1", '<')) {
+            $this->markTestSkipped('only for PHP > 7.1');
```

Comment:
Same as above but with the obviously different constraint, `@require PHP < 7.1`.